internal/order/sort.go

```diff
@@ -15,24 +17,17 @@ func ByID(n []graph.Node) {
 	sort.Slice(n, func(i, j int) bool { return n[i].ID() < n[j].ID() })
 }
 
-// BySliceValues sorts a slice of []int64 lexically by the values of the
-// []int64.
-func BySliceValues(c [][]int64) {
-	sort.Slice(c, func(i, j int) bool {
-		a, b := c[i], c[j]
-		l := len(a)
-		if len(b) < l {
-			l = len(b)
-		}
+// BySliceValues sorts a slice of []cmp.Ordered (such as []int64)
+// lexically by the values of the []cmp.Ordered.
```

Comment:
```suggestion
// BySliceValues sorts a slice of []cmp.Ordered lexically by the values of
// the []cmp.Ordered.
```

Reply:
Done! See the latest force-pushed commits.

I've also renamed `internal/order/sort.go` to `internal/order/order.go` for consistency.